README.md

```diff
@@ -1,17 +1,18 @@
-worker-vm
-=========
+# worker-vm
 
 [![test](https://github.com/eight04/worker-vm/actions/workflows/test.yml/badge.svg)](https://github.com/eight04/worker-vm/actions/workflows/test.yml)
 
 A `vm` library built on top of Deno's worker.
 
-Usage
-------
+## Usage
 
 ```ts
 import { VM } from "https://deno.land/x/worker_vm@v0.1.1/mod.ts";
 
-const vm = new VM(); // create a new VM Worker
+const vm = new VM({
+  timeoutMs: 30 * 1000,
+  permissionOptions: "none" // { "net": ["example.com:443"], "read": ["foo.txt", "bar.txt"]}
```

Comment:
I think we can rename `permissionOptions` to `permissions` since:

1. It's pretty obvious that it is an option.
2. `permissions` matches the identifier used by deno: https://docs.deno.com/api/web/~/WorkerOptions